app/views/layouts/registration.html.haml

```diff
@@ -3,7 +3,7 @@
     %meta{charset: 'utf-8'}/
     %meta{name: 'viewport', content: "width=device-width,initial-scale=1.0"}/
 
-    %title= content_for?(:title) ? "#{yield(:title)} - #{Spree::Config[:site_name]}" : "#{t(:welcome_to)} #{Spree::Config[:site_name]}"
+    %title= content_for?(:title) ? " #{yield(:title) } - #{Spree::Config[:site_name]}".html_safe : "#{t(:welcome_to)} #{Spree::Config[:site_name]}"
```

Comment:
That space shouldn't be at the start of the string, right?

Reply:
Yeah, a mistake, sorry.